### DISCUSSION TITLE
pantalla amarilla en elementary os 7.1

### DISCUSSION THREAD

**Discussion by germanfp28** [elementary/triage#373] [Q&A]
Cuando instalo elementary todo va correcto, cuando termina y se reinicia y me dice que quite el usb booteable y le doy a intro para seguir con la finalizacion de elemtary ya me sale en amarillo todo, he esto buscando sobre esto, y he intentado varias cosas pero es imposible, si alguien le ha pasado lo mismo y lo ha solucionado que me lo comente, GRACIAS

**Comment by EzelB7** [comment]
ME PASA LO MISMOOO, lo acabo de instalar y aparece muy calido o amarillo el inicio, pudiste resolverlo???????
